doc/00_intro.md

```diff
@@ -57,3 +57,28 @@ options:
 ```
 
 > 📝 The tool accepts five different settings for the logging level: 1 for error logging, 2 for warnings, 3 for info, 4 for debugging information and 5 for tracing. Tracing should not be used on large programs.
+
+
+### Targets
+
+The `taint` and `backtrace` tools of Argot support running against specific targets that can be defined in the config file (as opposed to passed in the command line).
+
+The targets are define in `targets`, and each target must have a `name` and a set of `files`. The file paths are taken relatively to the directory where the config file is by default, or relatively to the project root when the `option` `project-root` is defined. 
+For example, we can define two targets in the following config file:
+```yaml
+options:
+  project-root: "../"
+targets:
+  - name: "target1"
+    files:
+      - "cmd/target1/main.go"
+  - name: "target2"
+    files:
+      - "cmd/target2/main.go"
+
+```
+This assumes the config file is in a directory next to the `cmd` directory that contains all the executables of the project.
+Analysis problems that can use specific targets will have a `targets: ["target1", "target2"]` option for example. Then each tool can be run with a single argument, the config file.
+If some file paths are specified on the command line, the targets are ignored. The `"all"` target is a special target that means the problem applies to every target.
```

Comment:
What happens if the targets in the config differ from the ones passed in the command line?

Reply:
If some files are passed on the command line, then the tools behave like before: all the analyses run on the target provided on the command line (the targets in the config file have no effect).

Reply:
Should we error or display a warning if that happens? It might confuse the user otherwise